code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -301,33 +302,40 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
-			else
-				. += "Advisory Level: <b>Green Star</b></center><BR>"
-				. += "Your sector's advisory level is Green Star. Surveillance information shows no credible threats to Nanotrasen assets within the Spinward Sector at this time. As always, the Department advises maintaining vigilance against potential threats, regardless of a lack of known threats."
-		if(20 to 39)
-			. += "Advisory Level: <b>Yellow Star</b></center><BR>"
-			. += "Your sector's advisory level is Yellow Star. Surveillance shows a credible risk of enemy attack against our assets in the Spinward Sector. We advise a heightened level of security, alongside maintaining vigilance against potential threats."
-		if(40 to 65)
-			. += "Advisory Level: <b>Orange Star</b></center><BR>"
-			. += "Your sector's advisory level is Orange Star. Upon reviewing your sector's intelligence, the Department has determined that the risk of enemy activity is moderate to severe. At this advisory, we recommend maintaining a higher degree of security and alertness, and vigilance against threats that may (or will) arise."
-		if(66 to 79)
-			. += "Advisory Level: <b>Red Star</b></center><BR>"
-			. += "Your sector's advisory level is Red Star. The Department of Intelligence has decrypted Cybersun communications suggesting a high likelihood of attacks on Nanotrasen assets within the Spinward Sector. Stations in the region are advised to remain highly vigilant for signs of enemy activity and to be on high alert."
-		if(80 to 99)
-			. += "Advisory Level: <b>Black Orbit</b></center><BR>"
-			. += "Your sector's advisory level is Black Orbit. Your sector's local comms network is currently undergoing a blackout, and we are therefore unable to accurately judge enemy movements within the region. However, information passed to us by GDI suggests a high amount of enemy activity in the sector, indicative of an impending attack. Remain on high alert, and as always, we advise remaining vigilant against any other potential threats."
-		if(100)
-			. += "Advisory Level: <b>Midnight Sun</b></center><BR>"
-			. += "Your sector's advisory level is Midnight Sun. Credible information passed to us by GDI suggests that the Syndicate is preparing to mount a major concerted offensive on Nanotrasen assets in the Spinward Sector to cripple our foothold there. All stations should remain on high alert and prepared to defend themselves."
+	if (prob(PULSAR_REPORT_CHANCE))
+		. += "Advisory Level: <b>Pulsar Star</b></center><BR>"
+		. += "Your sector's advisory level is Pulsar Star. A large unknown electromagnetic field has stormed through nearby surveillance equipment. No surveillance data has been able to be obtained showing no credible threats to Nanotrasen assets within the Spinward Sector. The Department advises maintaining high alert against potential threats, regardless of a lack of information."
+	else
```

Comment:
I don't really like this because of how ugly the indentation gets, but we also can't early return here either because this logic is in the middle of an important proc? that's really stupid, just spin this out into a brand new proc like `generate_command_report_fluff()` so we can use early returns to our heart's content and not end up returning out of really important gamemode stuff